tests/cosmogony_lux_test.rs

```diff
@@ -87,3 +87,26 @@ fn test_lux_admin_levels_with_serialisation() {
 
     test_wrapper_for_lux_admin_levels(cosmogony_from_json);
 }
+
+#[test]
+fn test_lux_zone_types() {
+    // Check the zone types in the built cosmogony
+    let cosmogony = create_cosmogony_for_lux();
```

Comment:
even if that's way better to separate the tests, isn't the cosmogony rebuilding a bit too long ?

Rust definitely lacks a good py.test like framework 😜

Reply:
```
real	0m10.105s
user	0m30.044s
sys	0m0.508s
```